debian/control

```diff
@@ -51,13 +51,13 @@ Build-Depends: dbus-test-runner,
                libqt5sql5-sqlite,
                xvfb,
                cmake,
-               pep8,
-               python3-pep8,
-               python3-flake8 (>= 2.2.2-1ubuntu4) | python3-flake8:native,
+               pep8 <!nocheck>,
+               python3-pep8 <!nocheck>,
+               python3-flake8:native (>= 2.2.2-1ubuntu4) | python3-flake8:native,
```

Comment:
Hrmm. Both sides of this or dependency are the same, and I'm not sure flake8 is used anyway, so this dep should be removed. Instead perhaps having `python3-pyflakes <!nocheck>` would be better (as it's what is being used by the tests).